Example/PaymentSheet Example/PaymentSheet Example/ExampleWalletButtonsView.swift

```diff
@@ -509,11 +509,12 @@ class ExampleWalletButtonsModel: ObservableObject {
 
     var shopPayConfiguration: PaymentSheet.ShopPayConfiguration {
         let singleBusinessDay = PaymentSheet.ShopPayConfiguration.DeliveryEstimate.DeliveryEstimateUnit(value: 1, unit: .business_day)
```

Comment:
Do we not support next day? Maybe we should have the example use the string-based delivery estimate? https://docs.stripe.com/js/elements_object/create_express_checkout_element#express_checkout_element_create-options-shippingRates-deliveryEstimate

Reply:
Aie! thank you for calling this out -- this API needs to be re-worked!

Reply:
<img width="398" alt="Screenshot 2025-07-02 at 9 53 51 PM" src="https://github.com/user-attachments/assets/0fb7fe1d-63c0-44c0-8750-29dd4ed73c75" />

Demonstrates all different cases -- the "NaN to Nan business days" is assumed to be a bug in the wallet UI's code

Reply:
Seems good! Let's flag NaN to NaN to the owning team